CHANGELOG.rst

```diff
@@ -13,6 +13,10 @@ Added:
 
 - Ability to interpolate stimulator output waveforms from start timepoints of each subprotocol
 
+Changed:
```

Comment:
This PR fixes a bug, so the changelog entry should be under `Fixed`, not `Changed`